sdk/react/src/lib/fragment-components/a-popover.tsx

```diff
@@ -0,0 +1,282 @@
+import React from 'react';
+import {
+	Popover,
+    PopoverContent,
+	Dropdown,
+	TextInput,
+    Checkbox
+} from '@carbon/react';
+import { css, cx } from 'emotion';
+import { AComponent, ComponentInfo } from './a-component';
+
+import image from './../assets/component-icons/popover.svg';
+import {
+	angularClassNamesFromComponentObj,
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj
+} from '../helpers/tools';
+import { styleObjectToString } from '@carbon-builder/player-react';
+
+export const APopOverSettingsUI = ({ selectedComponent, setComponent }: any) => {
+
+    const alignItems = [
+        {id: 'top', text: 'Top'},
+        {id: 'right', text: 'Right'},
+        {id: 'bottom', text: 'Bottom'},
+        {id: 'left', text: 'Left'}
+    ];
```

Comment:
There are way more align items than this. See https://github.com/carbon-design-system/carbon/blob/b3488785126adecd6709370d2bd7786133d83213/packages/react/src/components/Popover/index.tsx#L48